exercises/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,18 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+function uppercaseName(mentorName) {
+  return mentorName.toUpperCase();
+}
+
+function greetingMaker(mentorName) {
+  let greeting = "HELLO " + uppercaseName(mentorName);
```

Comment:
I think, your function 'greetingMaker' can have two parameters, and the reason for that is what if we wanted to change the word 'HELLO' to something else later in future. And to do that you can write it like this :

```
function greetingMaker(capitalGreeting, name){
    let greeting = capitalGreeting.toUpperCase() + " " + name
    return greeting;
};
```

Reply:
You're absolutely right. I will change it. 
Thank you for the feedback